precompiles/distribution/integration_test.go

```diff
@@ -609,8 +609,8 @@ var _ = Describe("Calling distribution precompile from EOA", func() {
 			// get base fee to use in tx to then calculate fee paid
 			bfQuery, err := s.grpcHandler.GetBaseFee()
```

Comment:
This is the feemarket module's base fee. We should use the one on the evm module because these are ethereum txs